applications/vessel-history/src/features/home/Home.tsx

```diff
@@ -107,16 +109,23 @@ const Home: React.FC<LoaderProps> = (): React.ReactElement => {
           </div>
         )}
         {query && (
-          <div>
-            {searching && <Spinner className={styles.loader}></Spinner>}
-            {!searching && (
-              <div className={styles.offlineVessels}>
-                {vessels.map((vessel, index) => (
-                  <VesselListItem key={index} vessel={vessel} />
-                ))}
-              </div>
-            )}
-          </div>
+          <Fragment>
+            <ul className={styles.searchResults}>
+              {searching && (
+                <SearchPlaceholder>
+                  <Spinner className={styles.loader}></Spinner>
+                </SearchPlaceholder>
+              )}
+              {!searching && vessels.length > 0 && (
+                <div className={styles.offlineVessels}>
+                  {vessels.map((vessel, index) => (
+                    <VesselListItem key={index} vessel={vessel} />
+                  ))}
+                </div>
+              )}
+              {!searching && vessels.length == 0 && <SearchNoResultsState />}
```

Comment:
===

Reply:
@rdgfuentes are you using vscode? because it should warning you of using `==`

Reply:
Yes, but in the rush I missed the warning. Fixed now